docs/manuals/auth/index.md

```diff
@@ -48,28 +46,30 @@ Example of configuring LDAP server connection in Taipy:
     auth_enabled = True
     ```
 
-## Roles
+# Roles
+
+Taipy has four predefined user roles names that can be assigned to users. Each of these predefined roles provide a different set of capabilities and will be described in details below.
 
-Taipy has 4 roles to define a user. Each with different capabilities and will be described in detail below.
+## TAIPY_READER
```

Comment:
If all those 'TAIPY_role' are string, I guess you should document them as such:

```## "TAIPY_READER"```
as well as everywhere in the doc content.